src/Simulation/TargetDefinitions/TargetPackages/Type3.Package.props

```diff
@@ -18,7 +18,7 @@
     <QsharpCompile Include="..\TargetDefinitions\Intrinsic\ApplyUncontrolledX.qs" />
     <QsharpCompile Include="..\TargetDefinitions\Intrinsic\ApplyUncontrolledY.qs" />
     <QsharpCompile Include="..\TargetDefinitions\Intrinsic\ApplyUncontrolledZ.qs" />
-    <QsharpCompile Include="..\TargetDefinitions\Intrinsic\MZ.qs" />
+    <QSharpCompile Include="..\TargetDefinitions\Intrinsic\MZ.qs" />
```

Comment:
The other elements list it as `QsharpCompile` (with a little `s`). Should the others be updated as well to capital `S` for consistency?